config/manager/manager.yaml

```diff
@@ -63,6 +63,7 @@ spec:
             - name: KEDA_HTTP_DEFAULT_TIMEOUT
               value: ""
           securityContext:
+            runAsNonRoot: true
```

Comment:
I'm not sure if we should sneak this in this PR. Might be better to do in another PR + changelog entry?

Reply:
this is just a minor change because the policy is already defined at pod level, I added it to solve a semgrep issue that requires is also at container level